docs/platform/howto/disk-autoscaler.rst

```diff
@@ -0,0 +1,287 @@
+Scale your Aiven service disks automatically
+============================================
+
+Discover the service disk autoscaler and its capabilities. Find out how it works and how to use it with your Aiven services.
+
+.. topic:: Pricing
+
+   Costs of using disk autoscaler depend on your service type and plan. You're only charged based on actual storage usage on your service. For more details on the costs of using disk autoscaler, check the same for dynamic disk sizing (DDS) in `Aiven Plans and Pricing <https://aiven.io/pricing?product=kafka>`_ or contact us at sales@Aiven.io.
+
+Why use disk autoscaling
+------------------------
+
+Service disk autoscaler increases disk storage capacity automatically when the disk is running out of space. Disk autoscaling allows you to improve the cost-efficiency of operating your Aiven services: you can start with a relatively small-sized disk and only have it scaled up when needed with no risk of running out of disk space at any point.
+
+How it works
+------------
+
+There are a few steps illustrating how disk autoscaler works:
+
+1. You create an disk autoscaler integration endpoint in your Aiven project setting the maximum additional storage at the same time.
+2. You enable a disk autoscaler integration for your service using the new disk autoscaler integration endpoint.
+3. From that point onward, the evaluation of disk space availability for your service is done every 30 seconds.
+4. When disk storage consumption reaches the threshold for a specific service, disk autoscaler increases available storage space by 10% every time taking the currently-used service plan as a baseline.
```

Comment:
This is a bit misleading. We increase 10% of the current disk-size every time, we don't increase 10% of the base-plan.

i.e:

1. Disk size is 500GB
2. Disk runs full -> We add 10% = 50GB. Total disk size is now 550GB
3. Disk runs full again -> We add 10% = 55GB. Total disk size is now 605GB.

etc